variables.tf

```diff
@@ -64,6 +64,12 @@ variable "lambda_policy_json" {
   default     = null
 }
 
+variable "create_lambda_policy" {
```

Comment:
Would suggest to rename the variable to `lambda_attach_policy_json`:
```suggestion
variable "lambda_attach_policy_json" {
```

Reply:
fair. changed it.